public/app/features/templating/template_srv.ts

```diff
@@ -347,6 +349,36 @@ export class TemplateSrv implements BaseTemplateSrv {
     });
   }
 
+  getVariablesMapInTemplate(
+    target: string,
+    scopedVars: ScopedVars,
+    format?: string | Function
+  ): Record<string, VarValue> {
+    const regexp = new RegExp(this.regex);
+    const values: Record<string, VarValue> = {};
+
+    // see comments about regex variable for what the following parameters capture
```

Comment:
I'm not sure if this comment is super clear. I found it extremely helpful once I realized these parameter names match up with the comment in `public/app/features/variables/utils.ts` and I'd like to keep some sort of cross reference. Not sure if other people would find this useful or the best way to do it, however.

Reply:
That's a really good idea - done!